### DISCUSSION TITLE
Img wird nicht angezeigt im VSC aber in Codesandbox schon

### DISCUSSION THREAD

**Discussion by DennisWarchocki** [neuefische/questions#89] [Web] [answered]
Hallo Leute,

Ich sitze gerade an einem Gruppenprojekt und habe ein Problem. 
Ich wollte im Header ein Bild als Banner einfügen. Das Bild wird aber nicht bei mir angezeigt im Preview. 
Habe es dann im Codesandbox getestet dort geht es aber. Hat jemand eine Idee woran es liegt, oder wie ich das besagte Bild doch nutzen kann im VSC?

abload.de/img/rickandmortypicjlci4.png

**Comment by ahohnsen** [comment]
Hallo Dennis,

Bitte poste einen Link zum Projekt/Code, damit wir dir weiterhelfen können. 🙂

**Comment by pixelass** [comment, marked as answer]
Wir konnten das Problem mittlerweile lösen. `abload.de` hat das Bild wohl blockiert. Die Lösung war es das Bild lokal zu kopieren und dann zu verwenden `<img src="/images/this-is-the-image.jpg"/>`
